definitions/pipelines/detect_variants.cwl

```diff
@@ -62,6 +62,9 @@ inputs:
     docm_vcf:
         type: File
         secondaryFiles: [.tbi]
+    filter_docm_variants:
+        type: int?
```

Comment:
If this is being used as a boolean, it'd be better to either have it be a boolean in the workflow (and translated to `1`/`0` in the tool) or to be an enum so that `0` and `1` are the only allowed values.